[
    {
        "line": "51",
        "column": "11",
        "severity": "warning",
        "message": "Catching 'Exception' is not allowed.",
        "source": "com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatchCheck"
    },
    {
        "line": "54",
        "column": "32",
        "severity": "warning",
        "message": "The String \"Reason :\" appears 4 times in the file.",
        "source": "com.puppycrawl.tools.checkstyle.checks.coding.MultipleStringLiteralsCheck"
    },
    {
        "line": "79",
        "column": "11",
        "severity": "warning",
        "message": "Catching 'Exception' is not allowed.",
        "source": "com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatchCheck"
    },
    {
        "line": "108",
        "column": "11",
        "severity": "warning",
        "message": "Catching 'Exception' is not allowed.",
        "source": "com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatchCheck"
    },
    {
        "line": "138",
        "column": "11",
        "severity": "warning",
        "message": "Catching 'Exception' is not allowed.",
        "source": "com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatchCheck"
    }
]